src/bp-core/admin/bp-core-admin-schema.php

```diff
@@ -632,3 +626,61 @@ function bp_core_install_nonmember_opt_outs() {
 	 */
 	do_action( 'bp_core_install_nonmember_opt_outs' );
 }
+
+/**
+ * Members Notices installation function.
+ *
+ * If the `bp_messages_notices` table exists, we simply need to rename it.
+ *
+ * @since 15.0.0
+ */
+function bp_core_install_members_notices() {
```

Comment:
It would be useful only if the `wp_bp_messages_notices` table still exists, the upgrade routine is renaming it to `wp_bp_notices`. I'm unsure it's worthing the effort, but let's get team's opinion about it!